.env.example

```diff
@@ -1,4 +1,4 @@
-USERNAME=
+STEAMID=
```

Comment:
maybe, `STEAMUSERNAME=` will be better, since everyone will associate to steam64id

Reply:
agree, my friend just made the same mistake